### DISCUSSION TITLE
How to share a catalog

### DISCUSSION THREAD

**Discussion by donmbelembe** [netflie/whatsapp-cloud-api#72] [Q&A] [answered]
Hello, I would like to know if sending a catalog is already implemented has described here

https://developers.facebook.com/docs/whatsapp/cloud-api/guides/sell-products-and-services/share-products#get-started

**Comment by aalbarca** [comment, marked as answer]
Hi,

Interactive messages will be supported in 2.x version. This version is been testing currently.

Best regards.
